### DISCUSSION TITLE
Does now support (or plan to) private network ?

### DISCUSSION THREAD

**Discussion by lorenzofox3** [vercel/vercel#4457] [General] [answered]
I was wondering if it was possible to make some lambdas available only for another lambda and not the whole wild web ? :)

Use case example: a token inspection endpoint should only be accessible from a trusted API endptoint's middleware

**Comment by styfle** [comment, marked as answer]
Hi @lorenzofox3 

We have thought about doing this for cron jobs (#146) but its not a priority right now.

As a workaround, you could implement basic auth or jwt as a means to block traffic that should not access your function.

**Reply by lorenzofox3** [reply]
sounds good, thanks !
